tests/e2e_regression/eden/expected/blocks.body

```diff
@@ -1,603 +1,903 @@
 {
   "blocks": [
     {
+      "epoch": 28018,
+      "gas_limit": "0",
```

Comment:
Just checking - is this expected? Is the gas limit set dynamically, but in practice it's not set often?

Reply:
Block gas limit is still 0 for now on mainnet and will be updated soon with an upcoming governance proposal: https://github.com/oasisprotocol/oasis-core/blob/66af60621e63fc250d00bffe7e6f70d5075c9f69/go/upgrade/migrations/consensus_240.go#L59